packages/devtools_app/lib/src/screens/memory/memory_screen.dart

```diff
@@ -865,17 +868,17 @@ class MemoryBodyState extends State<MemoryBody>
         Theme.of(context).colorScheme.hoverSmallValueTextStyle;
 
     List<Widget> hoverPartImageLine(
-      String name, {
-      String image,
-      Color colorPatch,
+      String? name, {
+      String? image,
+      Color? colorPatch,
       bool dashed = false,
       double leftEdge = 5.0,
     }) {
-      String displayName = name;
+      String? displayName = name;
```

Comment:
this can be non-nullable once name is also non-nullable

Reply:
done